### PR TITLE
add unit test for cgroup-manager

### PR DESCRIPTION
**Pull request by yanxuean**
Signed-off-by: yanxuean <yan.xuean@zte.com.cn>

**What this PR does / why we need it**:
1. add two case for testing empty cgroupname and cgroupname without /.
2. add test case for get CgroupFsName.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```
/sig node

### CODE REVIEW

**Comment by yanxuean** [comment]
/assign @yujuhong 
PTAL

**Comment by yanxuean** [comment]
ping for approve @yujuhong 
thanks

File: pkg/kubelet/cm/cgroup_manager_linux_test.go [hunk 1]
```diff
@@ -87,14 +91,48 @@ func TestLibcontainerAdapterAdaptToSystemdAsCgroupFs(t *testing.T) {
 			input:    "/Burstable/pod_123",
 			expected: "Burstable.slice/Burstable-pod_123.slice/",
 		},
+		{
```
Comment (dashpole) [inline pkg/kubelet/cm/cgroup_manager_linux_test.go]: remove duplicated entry

**Review by dashpole — COMMENTED** [review]
can you provide a little more context? Is this required because of a refactor or something?

File: pkg/kubelet/cm/cgroup_manager_linux_test.go (hunk 1 above)
Comment (yanxuean) [reply pkg/kubelet/cm/cgroup_manager_linux_test.go]: The case test the cgroupName without underscore.

**Comment by yanxuean** [comment]
> can you provide a little more context? Is this required because of a refactor or something?

@dashpole 
1.add two case for testing empty cgroupname and cgroupname with /.
2.add test case for get CgroupFsName.

**Comment by yanxuean** [comment]
Thanks @dashpole .
@derekwaynecarr  PTAL, Thanks

**Comment by yanxuean** [comment]
/assign @yujuhong 
PTAL

File: pkg/kubelet/cm/cgroup_manager_linux_test.go (hunk 1 above)
Comment (derekwaynecarr) [reply pkg/kubelet/cm/cgroup_manager_linux_test.go]: when do we have a case without a fully qualified input path?

**Review by derekwaynecarr — CHANGES_REQUESTED** [review]
i do not fully understand the motivation for this pr.

File: pkg/kubelet/cm/cgroup_manager_linux_test.go [hunk 2]
```diff
@@ -87,14 +91,48 @@ func TestLibcontainerAdapterAdaptToSystemdAsCgroupFs(t *testing.T) {
...
+	testCases := []struct {
+		input    string
+		expected string
+	}{
+		{
+			input:    "/",
+			expected: "/",
+		},
+		{
+			input:    "Burstable.slice/",
```
Comment (derekwaynecarr) [inline pkg/kubelet/cm/cgroup_manager_linux_test.go]: is there a code path where we pass this in outside of tests?  i am not aware of any, and would worry it was sign of a bug.

**Comment by yujuhong** [comment]
/unassign
@derekwaynecarr is a better reviewer for this part of the code

**Pull request closed without merging**